db/config.c

```diff
@@ -966,16 +966,19 @@ static int read_lrl_option(struct dbenv *dbenv, char *line,
                     return -1;
                 }
             }
+            dyns_init_globals();
             rc = dyns_load_schema(fname, (char *)gbl_dbname, tblname);
             if (rc != 0) {
                 logmsg(LOGMSG_ERROR, "Error loading %s schema.\n", tok);
+                dyns_cleanup_globals();
```

Comment:
Same - I prefer if we don't sprinkle error handling everywhere.

Reply:
Agreed.